Public/Sdk/Public/Transformers/Transformer.Execute.dsc

```diff
@@ -219,6 +219,7 @@ namespace Transformer {
         hasUntrackedChildProcesses?: boolean;
         allowPreservedOutputs?: boolean;
         passThroughEnvironmentVariables?: string[];
+        preserveOutputWhitelist?: (File | Directory)[];

```

Comment:
Can you add a doc comment on what this does?
I had to read the code to infer the semantics as the name can imply both: "it should preserve output unless in this list list" or "it should purge unless it is in this list" as the term whitelist vs blacklist is dependent on the semantic :)

Reply:
agree, done!

---
In reply to: [287435027](https://github.com/microsoft/BuildXL/pull/369#discussion_r287435027) [](ancestors = 287435027)